src/stabilities/styles.tsx

```diff
@@ -0,0 +1,186 @@
+import { makeStyles } from '@material-ui/styles';
+import { getMinMediaQuery } from './utils';
+
+export const useGetStyles = () => {
+  const useStyles = makeStyles({
+    root: {
+      maxWidth: '100%',
+      height: 'auto',
+      display: 'flex',
+      borderRadius: '2px',
+      overflow: 'scroll',
```

Comment:
`overflow: scroll` will do this. Please read the docs to find the correct property for this usecase https://www.w3schools.com/cssref/pr_pos_overflow.asp
![image](https://user-images.githubusercontent.com/42913823/99614115-cde63380-2a53-11eb-8f20-4bf13a459eb7.png)

Reply:
replaced with 'auto'